hooks/pre_validations/12-check_capsule_tar.rb

```diff
@@ -4,12 +4,14 @@ def error(message)
   kafo.class.exit 101
 end
 
+ssl_directory = param('certs', 'ssl_build_dir')
+
 certs_tar = param('foreman_proxy_certs', 'certs_tar') ||
   param('foreman_proxy_content', 'certs_tar')
 
 if certs_tar.value
   certs_tar.value = File.expand_path(certs_tar.value)
-  if param('certs', 'deploy').value && !File.file?(certs_tar.value)
+  if param('certs', 'deploy').value && !File.file?(certs_tar.value) || !Dir.exist?(ssl_directory.value)
```

Comment:
Would the ssl_directory be created by this point in all cases? I am thinking of a fresh install.